src/plugins/remote-cache/storage/s3.ts

```diff
@@ -2,11 +2,32 @@ import s3 from 's3-blob-store'
 import aws from 'aws-sdk'
 
 export interface S3Options {
+  accessKey?: string
+  secretKey?: string
+  region?: string
   endpoint?: string
   bucket: string
 }
-export function createS3({ bucket, endpoint }: S3Options) {
+
+// AWS_ envs are default for aws-sdk
+export function createS3({
+  accessKey = process.env.AWS_ACCESS_KEY_ID,
+  secretKey = process.env.AWS_SECRET_ACCESS_KEY,
+  bucket,
+  region = process.env.AWS_REGION,
+  endpoint,
+}: S3Options) {
+  if (!accessKey || !secretKey || !(region || endpoint)) {
```

Comment:
This is incorrect as the credentials are not always provided using these environment variables.  You should not fall just because the environment variables are not provided, instead just don't give credentials to the library and let it try to find them in the various places it looks